extensions/amp-video/0.1/amp-video.js

```diff
@@ -40,9 +40,9 @@ import {isLayoutSizeDefined} from '../../../src/layout';
 import {listen, listenOncePromise} from '../../../src/event-helper';
 import {mutedOrUnmutedEvent} from '../../../src/iframe-video';
 import {
-  observeDisplay,
-  unobserveDisplay,
-} from '../../../src/utils/display-observer';
+  observeContentSize,
+  unobserveContentSize,
+} from '../../../src/utils/size-observer';
```

Comment:
Is there a PR with this already?

Reply:
Yeah. That existed for a long time now. All components (V0 and V1) that need an up-to-date size have already switched to this observer :)